src/allmydata/frontends/ftpd.py

```diff
@@ -317,7 +315,7 @@ def logout(): pass
 
 class FTPServer(service.MultiService):
     def __init__(self, client, accountfile, accounturl, ftp_portstr):
-        precondition(isinstance(accountfile, (unicode, NoneType)), accountfile)
+        precondition(isinstance(accountfile, (unicode, type(None))), accountfile)
```

Comment:
I definitely aim to minimize the impact on behavior, so I'll stick with this change for now, but I agree it may be preferable to use the idiomatic form separately.